### PR TITLE
readme.md updates

### PR DESCRIPTION
**Pull request by dlphesigns**
in this commit, i had:
1. fixed the dialect inconsistencies between American and British English;
2. improved some explanations for a better general understanding at a glance;
3. improved punctuation and capitalisation;
4. added a more serious nature to the document.

i think these updates will generally improve the main page for new visitors to come.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,94 +8,87 @@
 </p>
 
 
-### **Download the installer from the download link below or clone the repository to build :) Pre-Alpha 3 is out now and includes video support!**
+### **Pre-Alpha 3 is out now and includes video support! Download the installer from the download link below or clone the repository to build.**
 
-**Rise Media Player** brings all of your media to a whole new level. If it's videos, the music stored on your devices, discs or even your favourite streaming services; you'll love our player. One library for everything. Almost infinite customisability with settings that are second to none. Stream, browse and explore with Streaming Services - coming soon.
+**Rise Media Player** brings all of your media to a whole new level. If it's videos, the music stored on your devices, discs or even your favourite streaming services; you'll love our player. One library for everything. Almost infinite customisability with settings that are second to none. Stream, browse and explore your library. Vibe to music and watch your favourite clips!
 
-[Contribute to this Project](https://github.com/Rise-Software/Rise-Media-Player)
```
Comment (dlphin11) [inline readme.md:15]: improved capitalisation

File: readme.md [hunk 2]
```diff
@@ -8,94 +8,87 @@
 </p>
 
 
-### **Download the installer from the download link below or clone the repository to build :) Pre-Alpha 3 is out now and includes video support!**
+### **Pre-Alpha 3 is out now and includes video support! Download the installer from the download link below or clone the repository to build.**
 
-**Rise Media Player** brings all of your media to a whole new level. If it's videos, the music stored on your devices, discs or even your favourite streaming services; you'll love our player. One library for everything. Almost infinite customisability with settings that are second to none. Stream, browse and explore with Streaming Services - coming soon.
```
Comment (dlphin11) [inline readme.md:13]: removed repetition and added some spice to it

File: readme.md [hunk 3]
```diff
@@ -8,94 +8,87 @@
 </p>
 
 
-### **Download the installer from the download link below or clone the repository to build :) Pre-Alpha 3 is out now and includes video support!**
+### **Pre-Alpha 3 is out now and includes video support! Download the installer from the download link below or clone the repository to build.**
```
Comment (dlphin11) [inline readme.md:11]: improved order of ideas

File: readme.md [hunk 4]
```diff
@@ -8,94 +8,87 @@
...
+Created with **WinUI and the latest design ideologies**, **Rise Media Player** is modern while keeping all of the classic features people need. We use **WinUI 2.6/2.7** to keep our user interface, clean, modern and consistent while keeping the **Windows 11 experience**. Although, this app does work on Windows 10* too. We use all of your favourite materials: Mica and Acrylic. We use Microsoft's icons and controls as well as some of our own to create a generally native feeling experience. Our own controls and icons give users a truly personalised experience, being able to choose their own icon packs and with features like compact mode, you can use it on any Windows device!* More details below.
```
Comment (dlphin11) [inline readme.md:19]: little QOL improvements

File: readme.md [hunk 5]
```diff
@@ -8,94 +8,87 @@
...
+*The licence can also be reffered to as Terms and Conditions, or Terms of Service.
+
+### MSIX install instructions 
+**If Windows prompts about Untrusted Application**
+* Simply right-click the MSIX and click Properties;
+* Go to Digital Signatures tab;
+* Select Rise Software and click Details;
+* In next pop-up select View Certificate;
+* In next pop-up select Install Certificate;
+* In next step select Local machine (An UAC prompt may appear. Click Yes to continue);
```
Comment (itsWindows11) [inline readme.md:91]:
```suggestion
* In next step select Local machine (A UAC prompt may appear. Click Yes to continue);
```

File: readme.md [hunk 6]
```diff
@@ -8,94 +8,87 @@
...
+Created with **WinUI and the latest design ideologies**, **Rise Media Player** is modern while keeping all of the classic features people need. We use **WinUI 2.6/2.7** to keep our user interface, clean, modern and consistent while keeping the **Windows 11 experience**. Although, this app does work on Windows 10* too. We use all of your favourite materials: Mica and Acrylic. We use Microsoft's icons and controls as well as some of our own to create a generally native feeling experience. Our own controls and icons give users a truly personalised experience, being able to choose their own icon packs and with features like compact mode, you can use it on any Windows device!* More details below.
 
 ![Pre-Alpha 1 Render](https://user-images.githubusercontent.com/74561130/137600141-38c3a80a-519a-4e4f-9cae-7486b224e685.png)
 
-**Note**: This render may not contain all of the features shown in the app itself from GitHub, nor the Insider Previews. Some features are simply not coded or they are being tested internally. 
+**Disclaimer**: The actual product may not include all the functions featured in this mock-up. Some of them are being tested internally or planned to release later.
```
Comment (dlphin11) [inline readme.md:22]: what was that? it said that the mockup doesn't have all the features from the app??

File: readme.md [hunk 7]
```diff
@@ -8,94 +8,87 @@
...
+* Colourful icons setting for a Windows 11-like experience
 * Casting to devices, repeat
 * Insider Settings page with RiseMP wallpapers
-* Pick up where you left off support
-
-**Insider Exclusives**
-
-(You get these features if you sign up for the Insider Program and use the Store version. Cloning from GitHub will not contain these)
-* Nothing yet. There will be some exclusive features and themes for your PC though, so stay tuned! 
+* Pick up where you left off
```
Comment (dlphin11) [inline readme.md:44]: improved capitalisation

File: readme.md [hunk 8]
```diff
@@ -8,94 +8,87 @@
...
 [![OurTerms](https://user-images.githubusercontent.com/74561130/137586737-d792a57b-b2b0-410b-bb2a-a06a4da2ab52.png)](https://github.com/Rise-Software/Rise-Media-Player/blob/main/LICENSE)
 
 
-## Downloads, Contribute and the Insider Program
+## Downloads, Contribute and the Insider Programme
 
 MSIX packages are available now for the Pre-Alpha 3 build. We aren't going to be available in the Microsoft Store for a while as we will be releasing many builds over the next few months. Trying to approve each build on a regular basis is just not feasible. 
 
-Currently, all users can try out the MSIX packages. When we move to the store, you must be signed up to the insider program to be able to install. You can do this from the button below.
+Currently, all users can try out the MSIX packages. When we move to the Store, you must be signed up to the Insider Programme to be able to install. You can do this below!
```
Comment (dlphin11) [inline readme.md:63]: fixed american-british inconsistency

File: readme.md [hunk 9]
```diff
@@ -8,94 +8,87 @@
...
-* [**Omar Salas (@YourOrdinaryCat)**](https://github.com/yourordinarycat): **Programming & Development.**
+* [**Joseph Beattie (@josephbeattie)**](https://github.com/josephbeattie): **Rise Team Leader and Founder - Design, Occasional Programming, Development. Languages and Translations: English & Spanish**.
+* [**Omar Salas (@YourOrdinaryCat)**](https://github.com/yourordinarycat): **Programming & development.**
 
 **Contributors**
 
 * [**Agnihotra Nath (@AlfarexGuy2019)**](https://github.com/alfarexguy2019)
 * [**Makesh Vineeth (@MakeshVineeth)**](https://github.com/makeshvineeth)
-* [**Rareș Pița (@dlphin)**](https://github.com/dlphin)
+* [**Rareș (@dlphin11)**](https://github.com/dlphin11)
```
Comment (dlphin11) [inline readme.md:78]: corrected this ahah

File: readme.md [hunk 10]
```diff
@@ -8,94 +8,87 @@
...
+**If Windows prompts about Untrusted Application**
+* Simply right-click the MSIX and click Properties;
+* Go to Digital Signatures tab;
+* Select Rise Software and click Details;
+* In next pop-up select View Certificate;
+* In next pop-up select Install Certificate;
+* In next step select Local machine (An UAC prompt may appear. Click Yes to continue);
+* Select Place all certificates in Following store and click browse select Trusted People and click OK and Next;
+* Click Finish;
+* Confirmation windows will appear. Now, try installing the .msixbundle!
```
Comment (dlphin11) [inline readme.md:94]: better capitalisation

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
Looks good so far, but there's one slight typo here

File: readme.md (hunk 5 above)
Comment (itsWindows11) [inline readme.md]:
```suggestion
* In next step select Local machine (A UAC prompt may appear. Click Yes to continue);
```

**Comment by dlphesigns** [comment]
> Looks good so far, but there's one slight typo here

isn't it mandatory for the article to be 'an' when you have a vocale succeeding it?

**Comment by itsWindows11** [comment]
> > Looks good so far, but there's one slight typo here
> 
> isn't it mandatory for the article to be 'an' when you have a vocale succeeding it?

I don't think using 'an' applies to abbreviations. Also maybe don't start a review if you are just commenting on a snippet (use commit titles instead).

**Comment by itsWindows11** [comment]
@dlphin11 Any updates?

**Pull request closed without merging**